packages/backend/src/common/util.ts

```diff
@@ -47,3 +48,9 @@ export const stringifyCoordinates = (coordinates: LonLat): string =>
   `${coordinates.lon.toFixed(COORDINATES_FRACTION_DIGITS)}, ${coordinates.lat.toFixed(COORDINATES_FRACTION_DIGITS)}`;
 
 export const keyfy = (params: TileParamsWithKit): string => `${TILE_DETAILS_KEY_PREFIX}:${params.kit}:${params.z}/${params.x}/${params.y}`;
+
+export const hashValue = (value: unknown): string => {
+  const hash = createHash('md5');
```

Comment:
MD5 hash as been proven to be vurnable to a collision attack. We might have to look for another hash function.

Reply:
done.